.github/workflows/ci-idris2.yml

```
@@ -201,7 +201,7 @@ jobs:
       - name: Get Chez Scheme
         run: |
           git clone --depth 1 https://github.com/cisco/ChezScheme
-          c:\msys64\usr\bin\bash -l -c "pacman -S --noconfirm tar make"
+          c:\msys64\usr\bin\bash -l -c "pacman -S --noconfirm tar make mingw-w64-x86_64-gmp"
```

```suggestion
          c:\msys64\usr\bin\bash -l -c "pacman -S --noconfirm tar make mingw-w64-x86_64-gcc"
```

GCC depends on GMP. So if `mingw-w64-x86_64-gmp` is missing, we need to install `mingw-w64-x86_64-gcc`.
Add it for windows-self-host-racket too, line 419.

BTW, the first warning gives a clue to this: `idris_net.c:211:3: warning: implicit declaration of function 'inet_ntop'; did you mean 'inet_ntoa'?` (normally there are no warnings).

---

Let's try that!

---

@gallais The other windows job will need it too. (pity I can't add a suggestion there too)